bundles/org.openhab.binding.paradoxalarm/src/main/resources/ESH-INF/thing/ip150connector.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="paradoxalarm"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<bridge-type id="ip150">
+		<label>Paradox IP150 Module Connector</label>
+		<description>Paradox IP150 module connector</description>
+
+		<channels>
+			<channel id="communicationCommand" typeId="command" />
+		</channels>
+
+		<config-description>
+			<parameter name="refresh" type="integer" min="1" max="600">
+				<label>Refresh Interval</label>
+				<description>Specifies the refresh interval in seconds.</description>
+				<default>5</default>
+			</parameter>
+			<parameter name="panelType" type="text" required="false">
+				<label>Panel Type</label>
+				<description>PanelType</description>
+			</parameter>
+			<parameter name="ip150Password" type="text" required="true">
+				<label>IP150 Password</label>
+				<description>Password for accessing IP150</description>
+				<context>password</context>
+			</parameter>
+			<parameter name="pcPassword" type="text" required="true">
+				<label>PC Password</label>
+				<description>PC Password (section 3012 of configuration)</description>
+				<default>0000</default>
+				<context>password</context>
+			</parameter>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>IP Address</label>
+				<description>IP address of IP150</description>

```

Comment:
Can this only be an ip or just a hostname?

Reply:
I never looked into the low level implementation of java sockets but I would expect that if it's resolvable hostname, it should also work...

Reply:
Me too